cpp/modules/math.gl/core/src/core.h

```diff
@@ -251,14 +251,29 @@ Vector4<coord>::Vector4(const Vector4<othercoord> &other)
       w{static_cast<coord>(other.w)} {}
 
 template <typename coord>
-Vector4<coord> vectorProduct(const Vector4<coord> &, const Vector4<coord> &);
+Vector4<coord> vectorProduct(const Vector4<coord> &, const Vector4<coord> &, const Vector4<coord> &);
```

Comment:
We should use trailing return type syntax just like the method implementation does

```suggestion
auto vectorProduct(const Vector4<coord> &v1, const Vector4<coord> &v2, const Vector4<coord> &v3) -> Vector4<coord> 
```

Reply:
+1

Reply:
This overload was removed per a following comment, but the rest of the methods were checked to have this trailing return type.